app/bundles/CoreBundle/Assets/css/app.css

```diff
@@ -11351,3 +11351,32 @@ code:hover > .copy-icon {
 .contained-list-item__action > * {
   pointer-events: all;
 }
+/* === Column Picker Modal Styles === */
+#column-picker-form {
+  max-width: 400px;
+  margin: 0 auto;
+}
+
+#column-picker-form .form-check {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 12px;
+  padding: 6px 0;
+  font-size: 1.3rem;
+}
+
+#column-picker-form .form-check-label {
+  margin-bottom: 0;
+  flex-grow: 1;
+  cursor: pointer;
+  font-weight: 600;
+}
+
+#column-picker-form .form-check-input {
+  margin-left: 30px;
+  flex-shrink: 0;
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+}
```

Comment:
The app.css file is generated from the less files. So this must be moved there otherwise it will be removed next time someone will rebuild this file. See https://contribute.mautic.org/contributing-to-mautic/developer/local-environment-setup#development-environment.